Added AIDS, IT, MECH, and EIE syllabus based on previous VRSEC structure

afreenaiml opened the pull request:
Added the syllabus PDFs and semester-wise folder structure for the following branches under VRSEC_AP:

Artificial Intelligence and Data Science (AIDS)

Information Technology (IT)

Mechanical Engineering (MECH)

Electronics and Instrumentation Engineering (EIE)

Structure includes:
• Properly named folders for 1st to 8th semesters
• Syllabus PDFs placed inside each branch folder (based on previous VRSEC syllabus)

Let me know if anything needs to be changed!
Happy to contribute! 😊

Saumy1905 commented:
@afreenaiml You need to make PR as per the tasks specified

task 1.1 -> 1 branch syllabus
task 1.2 -> 3 branch syllabus
task 1.3 -> [3+] branch syllabus
task 2.1 -> 15 PYQs
task 2.2 -> 21 PYQs
task 2.3 -> 24 PYQs

- > task 1 => syllabus & project structure
- > task 2 => PYQ Addition

Labels
- > .1 => Beginner
- > .2 => Intermediate
- > .3 => Advanced

So, make sure to add atleast **15 PYQs**

Saumy1905 commented:
There is already a directory namely **VRSEC-AP**, so add accordingly 

![VRSEC-AP](https://github.com/user-attachments/assets/252b6335-0eec-4a10-885f-4aeba36c1af7)

Saumy1905 commented:
Also, from now on mention the task in your PR, so that we can directly hover to the task assigned to you which is affiliated to the PR

Ex: Issue: #16

afreenaiml commented:
@Saumy1905 I will make changes accordingly. Do i have to add 15PYQs also?

afreenaiml commented:
Ok, I will work on it.

afreenaiml commented:
📚 Tasks Covered:
🟦 Task 1.3 – [3+] Branch Syllabus (Advanced)
• Added syllabus and folder structure under the directory `VRSEC-AP` for the following branches:
   - AIDS
   - EIE
   - MTECH - STRUCTURAL ENGINEERING
   - MTECH - DATA SCIENCE
   - MTECH - CSE

📘 Structure:
• Properly organized folders by branch and semester
• Syllabus PDFs placed accordingly inside each semester folder

🟨 Task 2.3 – [24] PYQs (Advanced) 
• Created subject-wise folders in 1st, 2nd, and 3rd semesters of AIDS branch
• Added 24 PYQs following the correct naming convention:
   - Subject Name (Code) (Exam Month Year).pdf

📝 Notes:
- Followed the structure under existing directory `VRSEC-AP` as instructed
- Mentioning this PR for **Task 1.3 and Task 2.3**
- PR linked to the corresponding issue assigned: **Issue: #39**  

Happy to contribute! ✨

Saumy1905 commented:
@afreenaiml check this PR: https://github.com/Saumy1905/PYQFort/pull/163

You both are from the same college (VRSEC-AP) and doing the changes in the same directory

So, make sure to do accordingly to that
